client/views/network.tpl

```diff
@@ -1,5 +1,5 @@
 {{#each networks}}
-<section id="network-{{id}}" class="network" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
+<section id="network-{{id}}" class="network name-{{slugify name}} net-{{slugify serverOptions.NETWORK}}" data-id="{{id}}" data-nick="{{nick}}" data-options="{{toJSON serverOptions}}">
```

Comment:
I was thinking users' displayed network name, not the one server gives, but hm..